pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -314,10 +322,10 @@ func getMonitoringENV() []corev1.ServicePort {
 	}
 }
 
-// newRunnerPodForCR defines secondary resource #1 in same namespace as CR */
+// newRunnerPodForCR defines secondary resource #1 in same namespace as CR
 func newRunnerPodForCR(ce chaosTypes.EngineInfo) (*corev1.Pod, error) {
-	if len(ce.AppExperiments) == 0 || ce.AppUUID == "" {
-		return nil, errors.New("expected aExList not found")
+	if (len(ce.AppExperiments) == 0 || ce.AppUUID == "") && ce.Instance.Spec.ChaosType == "app" {
+		return nil, errors.New(" Application Experiment list not found")
```

Comment:
the error should be something like: `Application experiment list or UUID is empty`